generate/templates/templates/nodegit.cc

```diff
@@ -1,7 +1,7 @@
 // This is a generated file, modify: generate/templates/nodegit.cc.
```

Comment:
Probably. That would at least clarify a bit. Might be nice if we just do that when we get the compiled output back, like 
`// This is a generated file, modify: {root template}.` in the `generateNativeCode.js`